enginetest/evaluation.go

```diff
@@ -495,6 +495,8 @@ func WidenRow(sch sql.Schema, row sql.Row) sql.Row {
 			vw = uint64(x)
 		case float32:
 			vw = float64(x)
+		case float64:
+			vw = float64(float32(x))
```

Comment:
?

Reply:
There are rounding errors when using `float64`

Reply:
There really shouldn't be, this is a sign of a bug

Floats should round-trip with no precision loss

Does it come from parsing JSON?